### PR TITLE
Collector to rollingpin

### PR DESCRIPTION
**Pull request by krishnan-chandra**
👓  @spladug 

I think this should do it (verified that unit tests still work) but lmk if I missed anything.

### CODE REVIEW

File: events/__init__.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import base64
```
Comment (spladug) [inline events/__init__.py]: any reason to move this stuff? i think it'll be fine to live in the collector repo still and that keeps it more obviously separate from the injectors.

File: events/__init__.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+def health_check(request):
+    """A very simple health check endpoint."""
+    return {
+        "mood": u"\U0001F357",
+    }
+
+
+def make_wsgi_app(global_config, **settings):
```
Comment (spladug) [inline events/__init__.py]: baseplate entrypoint shouldn't have the kwargs stuff. global_config can just be called app_config.

File: events/__init__.py (hunk 1 above)
Comment (krishnan-chandra) [inline events/__init__.py]: No reason to move, it just appears to be the standard in other repos to keep the setup code in `__init__.py`. I can leave it in the collector if that's preferred.

Comment (spladug) [inline events/__init__.py]: yeah, they do __init__ just 'cus they're all single-service. this one's a bit more complex.

**Comment by krishnan-chandra** [comment]
:haircut_man:

**Pull request merged**